From: Robert Schwebel <r.schwebel@pengutronix.de>
Subject: [PATCH] fix QtScript handling for designer lib

FIXME: This patch does not work yet. The -lQtScript dependency
FIXME: leaks in via pkg-config!

While linking FIXME we get this error when compiling without support for
QtScript:

${toolchain}/bin/../lib/gcc/i586-unknown-linux-gnu/4.3.2/ ../../../../i586-unknown-linux-gnu/bin/ld: cannot find -lQtScript

Fix by adding QtScript only when supported.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 tools/designer/src/components/lib/lib.pro |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

Index: qt-embedded-linux-opensource-src-4.5.0/tools/designer/src/components/lib/lib.pro
===================================================================
--- qt-embedded-linux-opensource-src-4.5.0.orig/tools/designer/src/components/lib/lib.pro
+++ qt-embedded-linux-opensource-src-4.5.0/tools/designer/src/components/lib/lib.pro
@@ -64,7 +64,8 @@ PRECOMPILED_HEADER= lib_pch.h
 include(../../sharedcomponents.pri)
 include(../component.pri)
 
-unix:QMAKE_PKGCONFIG_REQUIRES = QtCore QtDesigner QtGui QtScript QtXml
+unix:QMAKE_PKGCONFIG_REQUIRES = QtCore QtDesigner QtGui QtXml
+unix:!isEmpty(QT_BUILD_SCRIPT_LIB):QMAKE_PKGCONFIG_REQUIRES += QtScript
 
 target.path=$$[QT_INSTALL_LIBS]
 INSTALLS        += target
